lib/src/session.dart

```diff
@@ -271,6 +272,7 @@ enum SyncErrorCategory {
 /// These errors will terminate the network connection
 /// (disconnect all sessions  associated with the affected connection),
 /// and the error will be reported via the connection state change listeners of the affected sessions.
+@Deprecated("Use SyncError or its subclasses instead.")
```

Comment:
This message is fine but the rest omit the `Use` word? This seems like copy/paste error

Reply:
Thanks for noticing that!